Makefile

```diff
@@ -98,6 +98,9 @@ generate: controller-gen
 	# we use this in pkg/controller/common/license
 	go generate -tags='$(GO_TAGS)' ./pkg/... ./cmd/...
 	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths=./pkg/apis/...
+	# Generate manifests e.g. CRD, RBAC etc.
```

Comment:
If we want to be closer to the kubebuilder default, they have a separate target called `manifests` which might be worthwhile, but I don't really feel strongly either way since I'm not sure we're ever going to want to run `generate` without `manifests`. One thing they do though is parameterize the trivial versions which might be worthwhile:
```
# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
CRD_OPTIONS ?= "crd:trivialVersions=true"
```